src/test/java/reposense/model/RepoConfigurationTest.java

```diff
@@ -44,7 +44,7 @@
     private static final Path WITHOUT_FORMATS_TEST_CONFIG_FILES = new File(RepoConfigurationTest.class.getClassLoader()
             .getResource("RepoConfigurationTest/repoconfig_withoutformats_test").getFile()).toPath();
     private static final Path GROUPS_TEST_CONFIG_FILES = new File(RepoConfigurationTest.class.getClassLoader()
-        .getResource("RepoConfigurationTest/repoconfig_groups_test").getFile()).toPath();
+            .getResource("RepoConfigurationTest/repoconfig_groups_test").getFile()).toPath();
```

Comment:
What is our required indentation level when we have multiple line wraps?

For example,
```
private static final Path GROUPS_TEST_CONFIG_FILES = new File(RepoConfigurationTest.class.getClassLoader()
            .getResource("RepoConfigurationTest/repoconfig_groups_test").getFile())
            .toPath();
```

Should the line `toPath();` be indented 8 spaces to the right relative to `.getResource(...` or  should it be on the same level as `.getResource(...` ?

With the new setting, checkstyle recommends that `toPath();` should be on the same level as `.getResource(...`
This seems ok to me.

Reply:
> Should the line `toPath();` be indented 8 spaces to the right relative to `.getResource(...` or should it be on the same level as `.getResource(...` ?

It should be 8 spaces to the right relative to the first line, aka `private static final...`